### ISSUE TITLE
[nl.klikaanklikuit] Push notification is not working with beta 4.0.13

### ISSUE THREAD

**Issue by nieuwe** [athombv/homey#2817]
**Metadata**

* **Homey Version**:
* **Homey Log Code**:
* **Smartphone Platform**:
* **Smartphone App Version**:

**If related to a Homey App:**
* **Homey App Version**:
* **Homey App Log Code**:
* **Affected Devices (preferably product codes)**:

**What did you try to achieve?**

**What did you expect as result?**

**What was the result?**

**Any other remarks**

**Comment by nieuwe** [comment]
I'm using klikaanklik uit V 4.0.13 and my homey has firmware 1.5.12
I made a flow with an ACDB 7000C remote and an ACDB700C chime.
When the remote is pushed I tell in the flow to send a pushmessage to my Iphone 8plus (IOS 11.4.1) but nothing happens. The Chime gives sound but no pushmessage
With version 4.0.11 of the homey app everything worked fine.
I've also have windowsensor AMST-606 and that sensor does everything fine. It sends a pushmessage to my Iphone when it is activated.

**Comment by Dijker** [comment]
Hi Nieuwe!

Push notifications and the [nl.klikaanklikuit]  app are not related, this cant be an issue.
there must be something wrong with your flows. 

(altough I must say that was a breaking change in the way the doorbell functions in BETA version 4.0.13)

From the App description: 
`What's new
v4.0.13
Changed doorbell devices to have a seperate device for the gong and the button. This fixes a issue where the device keeps triggering itself in a flow where both the gong and the button where used. If you're using the button in a flow, you'll have to add the button seperately and integrate it again in the flow.`

Could you please take another look at our contributing guidelines?
https://github.com/athombv/homey/blob/master/CONTRIBUTING.md

The more information Athom has, the easier it will be to reproduce the issue and provide a solution. If you are unable to do this or just want to receive regular support, please contact Athom through support@athom.com instead of using the bug tracker.

For help and support from the community you can ask on the Athom Community Forum https://community.athom.com/ or on Slack as a chat channel. Get your invite by entering your email address here and register for an Slack account. Login and join the conversation here: https://athomcommunity.slack.com

Can you please close this Issue with the button down here as this is no issue of valid feature request.

**Comment by nieuwe** [comment]
Hi Dijker

thnx for the very quick response!

Problem is solved! 

What happend is that I paired the remote and the chime with the remote. It's an option that is shown when you install the device in Homey. 
So when I installed the new version of the app the remote and the chime were directly paired.
What I've done to resolve the problem is remove both devices from my Homey and after that was done I reconnected them to Homey as a klikaanklikuit devices. I also made sure that they weren't connected together like devices connect in klikaanklikuit . (When I push the remote nothing happens)
Then I made a flow where the remote triggers the chime and sends a pushmessage to my phone using the newly paired devices.
Now everything works like a charm.

Maybe it's a good idea to let eveybody who wants to install the new beta version know to first remove the remote and chime, unpair them and install them as seperated, "clean" devices
